games-util/gamemode/gamemode-1.5.1.ebuild

```diff
@@ -25,11 +25,15 @@ SLOT="0"
 IUSE=""
 
 RDEPEND="
-	>=sys-apps/systemd-236[${MULTILIB_USEDEP}]
+	|| ( !sys-apps/systemd >=sys-apps/systemd-236[${MULTILIB_USEDEP}] )
```

Comment:
I wonder would it look better to depend either on dbus, or systemd?
If that's not wise, just make systemd a conditional dep and let's try with that. 

Or if you're 100 % sure it doesn't work at all without it, then unconditional dependency on systemd is ok I guess.

Are you sure the 32-bit lib needs 32-bit libsystemd?

Reply:
You're probably true: depending on either dbus or systemd should cover everything. Also, I think it actually doesn't need the multilib depend because only the daemon itself (64-bit) links to libsystemd.so, the 32-bit libs don't. But they link to 32-bit dbus, so we'd need a multilib dep on that?